sklearn/tests/test_impute.py

```diff
@@ -1009,9 +1025,22 @@ def test_missing_indicator_sparse_param(arr_type, missing_values,
 
     indicator = MissingIndicator(missing_values=missing_values,
                                  sparse=param_sparse)
+
+    if param_sparse is True and missing_values == 0:
+        with pytest.raises(ValueError):
+            X_fit_mask = indicator.fit_transform(X_fit)
+        return
+
+    if sparse.issparse(X_fit) and missing_values == 0:
+        with pytest.raises(ValueError):
+            X_fit_mask = indicator.fit_transform(X_fit)
+        return
+
     X_fit_mask = indicator.fit_transform(X_fit)
     X_trans_mask = indicator.transform(X_trans)
 
+
```

Comment:
remove on of the line

Reply:
ok, I removed all extra blank lines. Thanks!